src/zkevm_specs/evm_circuit/execution/error_oog_static_memory_expansion.py

```diff
@@ -0,0 +1,33 @@
+from zkevm_specs.util.param import N_BYTES_MEMORY_ADDRESS
+from ...util import FQ
+from ..instruction import Instruction
+from ..opcode import Opcode
+from ...util import N_BYTES_GAS
+
+
+def error_oog_static_memory_expansion(instruction: Instruction):
+    # retrieve op code associated to oog constant error
+    opcode = instruction.opcode_lookup(True)
+    is_mload, is_mstore, is_mstore8 = instruction.multiple_select(
+        opcode, (Opcode.MLOAD, Opcode.MSTORE, Opcode.MSTORE8)
+    )
+
+    # Constrain opcode must be one of MLOAD, MSTORE or MSTORE8.
+    instruction.constrain_equal(is_mload + is_mstore + is_mstore8, FQ(1))
+
+    # pop `offset`
+    offset = instruction.word_to_fq(instruction.stack_pop(), N_BYTES_MEMORY_ADDRESS)
+
+    # get total gas cost
+    constant_gas = FQ(3)
```

Comment:
will it better to use `GAS_COST_FASTEST` in param.py to replace this constant_gas cost

Reply:
fixed in [a84ae0a](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/474/commits/a84ae0a23eca01a64967e0ac59ef69429506899e)